<launch>
  <arg name="sigma_radial" default="0.2" />
  <arg name="sigma_polar" default="0.0" />
  <arg name="sigma_azimuthal" default="0.0" />
  <arg name="sigma_roll" default="0.0" />
  <arg name="sigma_pitch" default="0.0" />
  <arg name="sigma_yaw" default="0.0" />
  <arg name="plot_data" default="true" />

  <node pkg="tuw_noise" type="tuw_noise.py" name="tuw_noise">
      <param name="sigma_radial" value="$(arg sigma_radial)" />
      <param name="sigma_polar" value="$(arg sigma_polar)" />
      <param name="sigma_azimuthal" value="$(arg sigma_azimuthal)" />
      <param name="sigma_roll" value="$(arg sigma_roll)" />
      <param name="sigma_pitch" value="$(arg sigma_pitch)" />
      <param name="sigma_yaw" value="$(arg sigma_yaw)" />
      <param name="plot_data" value="$(arg plot_data)" />
      <remap from="marker" to="base_marker_detection" />
  </node>
</launch>

